etcd/etcd-live-cluster-reconfiguration.md

```diff
@@ -101,40 +101,34 @@ member 60e8a32b09dc91f1 is healthy: got healthy result from http://10.0.1.3:2379
 cluster is healthy
 ```
 
-We have to run `etcdctl` from a working node, or just use [`ETCDCTL_ENDPOINT`][etcdctl-endpoint] environment variable or parameter to specify `etcdctl` to use healthy member node.
+Run `etcdctl` from a working node, or use the [`ETCDCTL_ENDPOINT`][etcdctl-endpoint] environment variable or command line option to point `etcdctl` at any healthy member node.
 
-Then we have to [remove failed][etcdctl-member-remove] 10.0.1.2 member from your etcd cluster. This will tell all the other nodes in the cluster that a human has determined this node is dead and not to connect to it ever again:
+[Remove the failed member][etcdctl-member-remove] `10.0.1.2` from the etcd cluster. The remove subcommand informs all other cluster nodes that a human has determined this node is dead and not available for connections:
 
 ```sh
 $ etcdctl member remove 1609b5a3a078c227
 Removed member 1609b5a3a078c227 from cluster
 ```
 
-Then on the failed node (10.0.1.2) we have to stop etcd2 service:
+Then, on the failed node (`10.0.1.2`), stop the etcd2 service:
 
 ```sh
 $ sudo systemctl stop etcd2
 ```
 
-And clean up the `/var/lib/etcd2` directory:
+Clean up the `/var/lib/etcd2` directory:
 
 ```sh
 $ sudo rm -rf /var/lib/etcd2/*
 ```
 
-You have to make sure that this directory exists and empty. It you removed this directory accidentally, you can recreate it with the command below:
+Check that the `/var/lib/etcd2/` directory exists and is empty. If you removed this directory accidentally, you can recreate it with the proper modes by using:
 
 ```sh
 $ sudo systemd-tmpfiles --create /usr/lib64/tmpfiles.d/etcd2.conf
 ```
 
-or with this one:
-
-```sh
-$ sudo install -m 755 -d -o etcd -g etcd /var/lib/etcd2
```

Comment:
@kayrus I'm going to merge without adding the second way back; If you think it really needs to be there, let me know and we'll add the `install` invocation back. But I prefer to just update the doc should the etcd2.conf file change in future, and have just "one way to do it" in the doc.